docs/user-manual/invoking.tex

```diff
@@ -88,8 +88,8 @@ \section{Invoking the \pate{} Verifier}
   --read-only-segments ARG Mark segments as read-only (0-indexed) when loading
                            ELF
   --script FILENAME        Save macaw CFGs to the provided directory
-  --no-assume-stack-scope  Don't add additional assumptions about stack frame
-                           scoping
+  --assume-stack-scope     Add additional assumptions about stack frame 
+                           scoping during function calls (unsafe)
```

Comment:
While we're here, is there a brief description of what exactly this does and when the user would want to use it that we can add to the user manual? Maybe just a bit of elaboration at the end of `invoking.tex` framed as tips for invoking PATE, perhaps?